aws/resource_aws_wafv2_web_acl_test.go

```diff
@@ -49,29 +56,36 @@ func testSweepWafv2WebAcls(region string) error {
 			d.Set("lock_token", webAcl.LockToken)
 			d.Set("name", webAcl.Name)
 			d.Set("scope", input.Scope)
-			err := r.Delete(d, client)
 
-			if err != nil {
-				sweeperErr := fmt.Errorf("error deleting WAFv2 Web ACL (%s): %w", id, err)
-				log.Printf("[ERROR] %s", sweeperErr)
-				sweeperErrs = multierror.Append(sweeperErrs, sweeperErr)
-				continue
-			}
+			sweepResources = append(sweepResources, NewTestSweepResource(r, d, client))
 		}
 
 		return !lastPage
 	})
 
-	if testSweepSkipSweepError(err) {
-		log.Printf("[WARN] Skipping WAFv2 Web ACL sweep for %s: %s", region, err)
-		return sweeperErrs.ErrorOrNil() // In case we have completed some pages, but had errors
+	if err != nil {
+		errs = multierror.Append(errs, fmt.Errorf("error describing WAFv2 Web ACLs for %s: %w", region, err))
+	}
+
+	err = testSweepResourceOrchestrator(sweepResources)
+
+	// Since we cannot exclude Firewall Manager WebACLs from the sweepResources var above,
+	// we instead catch and ignore the following expected AccessDeniedException.
+	// Reference: https://github.com/hashicorp/terraform-provider-aws/issues/19149
+	if tfawserr.ErrMessageContains(err, "AccessDeniedException", "managed by Firewall Manager") {
+		return errs.ErrorOrNil()
```

Comment:
maybe we should log a warning before returning? though could get too verbose
e.g.
```
2021/06/17 14:08:45 [WARN] Skipping WAFv2 Web ACLs sweep for us-west-2: 2 errors occurred:
        * Error deleting WAFv2 WebACL: AccessDeniedException: You don't have permissions to delete the resource because it's managed by Firewall Manager.
        status code: 400, request id: c71575c0-fc32-469f-b6bf-7986000a3ef7
        * Error deleting WAFv2 WebACL: AccessDeniedException: You don't have permissions to delete the resource because it's managed by Firewall Manager.
        status code: 400, request id: 3bbcf3d3-38e9-40d6-8f57-5aac3366a1e7
```

Reply:
A nicer way to deal with this may be to add to `testSweepSkipSweepError` in `aws_sweeper_test.go`.

Reply:
hmm I didn't think about this before but WebACLs managed by Firewall Manager are prefixed with `FMManagedWebACLV2` so we _could_ skip over these when looping through the list of webACLs. what do you think of that as an alternative instead of waiting to catch the error within `testSweepSkipSweepError`?

Reply:
Makes sense. Looking for specific error text feels like it should always be the last resort.

Reply:
Addressed in 8c6b5da @ewbankkit